app/src/main/java/ai/elimu/vitabu/ui/storybook/StoryBookActivity.kt

```diff
@@ -69,7 +69,11 @@ class StoryBookActivity : AppCompatActivity() {
             }
 
             override fun onPageSelected(position: Int) {
-                timeSpent[position] = ((SystemClock.elapsedRealtime() - startReadingTime) / 1000).toInt()
+                val timeSpentOnPage = ((SystemClock.elapsedRealtime() - startReadingTime) / 1000).toInt()
```

Comment:
@tuancoltech I would add a brief comment here, explaining what this code does, so that understanding the logic will be easier.

Reply:
@jo-elimu I've just added one.